lib/index.ts

```diff
@@ -53,7 +53,7 @@ export function serviceWorkerFetchListener(): (e: FetchEvent) => boolean {
           timeout,
         }: ServiceWorkerReadRequest = await e.request.json();
         const data = earlyMessages[messageId];
-        if (data) {
+        if (messageId in earlyMessages) {
           delete earlyMessages[messageId];
           return success(data);
```

Comment:
```suggestion
          return success(earlyMessages[messageId]);
```

Thanks, this was a dumb mistake from me. How about getting rid of the `data` variable since it might not be used?